WebApiCoreSeed.WebApi/Startup.cs

```diff
@@ -92,15 +92,14 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
 
             //Enable swagger midleware
             app.UseSwagger();
-
+            
             // Enable middleware to serve swagger-ui (HTML, JS, CSS, etc.), specifying the Swagger JSON endpoint.
             app.UseSwaggerUI(c =>
             {
+                c.RoutePrefix = string.Empty;

```

Comment:
If you add a comment like `// Makes SwaggerUI the default/root route` you will become my new hero. The comment will be helpful for other devs because the line of code is not very clear.

Reply:
HAHAHAHAH you're right.